patches/net/minecraft/client/gui/screens/options/controls/KeyBindsScreen.java.patch

```diff
@@ -27,7 +27,7 @@
          if (this.selectedKey != null) {
 -            if (p_345810_ == 256) {
 +            var key = InputConstants.getKey(p_345810_, p_345447_);
-+            if (net.neoforged.neoforge.client.settings.KeyModifier.isKeyCodeModifier(key)) {
++            if (lastPressedModifier == InputConstants.UNKNOWN && net.neoforged.neoforge.client.settings.KeyModifier.isKeyCodeModifier(key)) {
```

Comment:
What does this change do? I'm not quite sure.

Reply:
Basically, this makes sure that the user is not currently holding a modifier key before assigning it to the last pressed key. Otherwise, if this was not checked, then the user would not be able to select two modifier keys at once.